robottelo/cli/factory.py

```diff
@@ -1288,6 +1372,90 @@ def make_host(options=None):
 
 
 @cacheable
+def make_fake_host(options=None, isolated_mode=False):
```

Comment:
At least it's not mandatory for both API & CLI - both interfaces can create fake host without specifying subnet. Remember we're talking about fake hosts, not real ones.